docs/src/main/sphinx/security/oauth2.rst

```diff
@@ -0,0 +1,106 @@
+========================
+OAuth 2.0 authentication
+========================
+
+Trino can be configured to enable OAuth 2.0 authentication over HTTPS for the
+Web UI and the JDBC driver. Trino uses the `Authorization Code
+<https://tools.ietf.org/html/rfc6749#section-1.3.1>`_ flow which exchanges an
+Authorization Code for a token. At a high level, the flow includes the following
+steps:
+
+#. the Trino coordinator redirects a user agent to the Authorization Server
```

Comment:
Use browser instead of `user agent`.  I know user agent is more accurate, but most people don't know that